### ISSUE TITLE
get clade and endemic data from matt kling and evaluate for incorporation

### ISSUE THREAD

**Issue by cschloss** [tnc-ca-geo/PLW#12]

**Comment by cschloss** [comment]
Matt Kling's Shiny site is here: https://matthewkling.shinyapps.io/phylodiversity/
It is good for exploring the data but if we want the finer res data we need to contact him directly

**Comment by cschloss** [comment]
Emailed Matt 2/11 asking for:
1. Combined richness (based on the 3 branches) but in a blank slate (no protection rolled in)
2. Endemism or RPE from Brett's work
3. Top 50 sites - for comparison to some of our prioritization when we get there
4. Marginal value based on species richness and endemism

**Comment by cschloss** [comment]
Hi Carrie,

I've just posted the high-resolution diversity layers online, at https://github.com/matthewkling/facets-of-phylodiversity/releases/tag/data810m

If you download the `diversity.zip` file, you'll see it contains 42 810m-resolution raster layers. The one called `species_D.tif` is species richness and should correspond the the dataset I previously sent you. There are also a bunch of endemism-related variables -- 4 for each of 6 diversity facets though they are not all unique. Endemism is the inverse of the California range size of a given taxon, where range size is the sum of presence probabilities across all grid cells. For each grid cell, we can calculate total endemism (E, the sum of taxon endemism scores weighted by their presence probability in the cell) and mean endemism (Em, the average of taxon endemism scores weighted by their presence probability in the cell). We also calculate phylogenetic versions of these metrics (PE and PEm, which are weighted by a taxon's branch length, i.e. unique evolutionary history, in addition to endemism). For a single metric that includes geographic rarity (endemism) and evolutionary rarity (phylogenetic branch length) and measures overall diversity (as opposed to the rarity of the mean resident taxon), I would go with `chronogram_PE.tif`. Hope that's not too confusing -- let me know what questions you have!

As I mentioned in my last email, the actual conservation prioritization results presented in Figure 3 of the paper can't easily be produced at 810m due to computational limits of the fairly basic optimization algorithm that we used. I now see that your requests A and C were both asking about those data. Since you are doing your own prioritization from the ground up, it might make more sense to use the raw input data rather than the outputs of our prioritization anyway. Happy to chat more about your goals and how we might be able to help. 

Cheers,
Matt
